civis/resources/_resources.py

```diff
@@ -146,15 +146,20 @@ def deprecated_notice(deprecation_warning):
     return "Deprecation warning!\n------------------\n" + deprecation_warning
 
 
-def doc_from_responses(responses):
+def doc_from_responses(responses, is_iterable):
     """ Return a doc string element from a responses object. The
     doc string describes the returned objects of a function.
     """
     response_code, response_object = next(iter(responses.items()))
     schema = response_object.get('schema', {})
     properties = get_properties(schema)
     if properties:
-        result_doc = "\n".join(docs_from_properties(properties))
+        if is_iterable:
+            resp_type = "civis.response.PaginatedResponse\n"
+        else:
+            resp_type = "civis.response.Response\n"
```

Comment:
I was compiling the Sphinx docs from this branch and saw `civis.response.Response` show up as plain text. If you put in (I need back ticks to actually show here and don't want to fight with Markdown rendering):

```
:class:`civis.response.Response`
```

then you should be able to turn it into a hyperlink that points to wherever the `Response` class's docstring appears in the compiled Sphinx docs. Same thing for `civis.response.PaginatedResponse`.

Reply:
great idea